digits/utils/errors.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2014-2015, NVIDIA CORPORATION.  All rights reserved.
+
+# This class contains all the DIGITS user-defined exceptions
+
+class DigitsError(Exception):
+    def __init__(self, value):
+        self.value = value
+    def __str__(self):
+        return self.value
+
+class DeleteError(DigitsError):
+    def __init__(self, value):
+        DigitsError.__init__(self, value)
```

Comment:
This is nice - I'll start using these in my code, too!

You don't really need any of that code in there. Just use this and treat them like normal exceptions:

```python
class DigitsError(Exception):
    pass
class DeleteError(DigitsError):
    pass
```